inc/widgets/largo-taxonomy-list.php

```diff
@@ -53,24 +87,27 @@ function widget( $args, $instance ) {
 				});
 			</script>
 
-		<?php } else { 
+		<?php
+		/*
+		 * Not the dropdown option
+		 */
+		} else {
 			echo '<ul class="' . $instance['taxonomy'] . '">';
 
-			$cat_args['title_li'] = '';
-			$tax_items = get_categories($cat_args);
+			$tax_items = get_categories($term_args);
 
-			switch ($instance['taxonomy']) {
+			switch ( $instance['taxonomy'] ) {
 				case 'series':
-					$this->render_series_list($tax_items, $instance);
+					$this->render_series_list( $tax_items, $instance );
 					break;
 				case 'category':
```

Comment:
this seems unnecessary, shouldn't we just use an if/else here to use the one function for series and the other function for everything else?